### PR TITLE
Add TLS and static token authentication. Add kratos for managing app

### PR DESCRIPTION
**Pull request by aatarasoff**
Addresses issue #5

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -2,88 +2,57 @@ package main
...
-		}
-	}()
-
-	err = startDebugServer(config.Cfg.Server.Debug.Addr)
+	defer pool.Close()
+
+	registryRepository := data.NewRegistryRepository(pool, logger)
+	registryServer := server.NewRegistryServer(registryRepository, logger)
+
+	app := kratos.New(
```
Comment (s-yakubovskiy) [inline cmd/main.go:41]: So we are using Kratos? ;)

File: docker-compose.yml [hunk 2]
```diff
@@ -17,20 +17,30 @@ services:
       timeout: 10s
       retries: 5
   pbuf-registry:
-    image: ghcr.io/pbufio/registry:v0.1.2
+    image: ghcr.io/pbufio/registry:v0.3.0-rc.1
```
Comment (s-yakubovskiy) [inline docker-compose.yml:20]: how tags are managed here? manual?

**Review by s-yakubovskiy — APPROVED** [review]
LGTM overall

File: docker-compose.yml (hunk 2 above)
Comment (aatarasoff) [reply docker-compose.yml:20]: Manual at the moment

File: cmd/main.go (hunk 1 above)
Comment (aatarasoff) [reply cmd/main.go:41]: Yep. The best for now. If you can suggest smth else that has reusable middleware for both http/grpc - it would be cool 😎

**Pull request merged**